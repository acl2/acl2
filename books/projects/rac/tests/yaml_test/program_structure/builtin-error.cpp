#include <array>
using namespace std;

// RAC begin

int foo(array<int, 3> a) { return abs(a); }
