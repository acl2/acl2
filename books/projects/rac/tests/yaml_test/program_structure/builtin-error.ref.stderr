builtin-error.cpp:7 (13-14): Expected `int`, got `int[3]`
 7 |   return abs(a);
   |              ^
note: Defined here: 
<builtin>
